Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,23 @@
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
```

Comment:
Hi everyone, sngrep developer here.

I have some commits to be merged that fix some ui problems with small terminals. They are not critical, so do you prefer not to include them in 0.2.2 and close the release?

Otherwise, 0.2.2 will include them and be closed on tuesday 10th (if no mayor bug appears).